DomainResponse.js

```diff
@@ -4,7 +4,7 @@ class DomainResponse {
   constructor(data) {
     return {
       domainName: data.distributionDomainName || data.regionalDomainName,
-      hostedZoneId: data.distributionHostedZoneId || data.regionalHostedZoneId,
+      hostedZoneId: data.distributionHostedZoneId || data.regionalHostedZoneId || 'Z2FDTNDATAQYW2',
```

Comment:
It might make more sense to pull this out as a constant and add a comment with the docs link you shared.